python/py-frozendict/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        Marco-Sulla python-frozendict 2.3.8 v
+
+name                py-frozendict
+version             2.3.8
+revision            0
+categories-append   devel
+license             LGPL-3
+supported_archs     noarch
+platforms           {darwin any}
+
+maintainers         nomaintainer
+
+description         frozendict is a simple immutable dictionary
+
+long_description    frozendict is a simple immutable dictionary. \
+                    It's fast as dict, and sometimes faster! \
+                    Unlike other similar implementations, \
+                    immutability is guaranteed: you can't change \
+                    the internal variables of the class, and they \
+                    are all immutable objects.
+
+checksums           rmd160  274fb504566cc57b1e5e9cac20a6c88822e77a21 \
+                    sha256  dc0629a5ac1e3f455b16fc781c3e0192ac64804b201be836866d6126d9470321 \
+                    size    310402
+
+python.versions     38 39 310 311
+python.pep517       yes
+python.pep517_backend setuptools
```

Comment:
last tao lines are the default for Python 3.7+ so they can be removed

Reply:
Done!